### PR TITLE
docs(apps_supported): Add files_lock

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

File: admin_manual/installation/apps_supported.rst [hunk 1]
```diff
@@ -35,6 +35,7 @@ Nextcloud Files
 * `Full Text Search - Elasticsearch Platform <https://github.com/nextcloud/fulltextsearch_elasticsearch>`_
 * `Full Text Search - Files <https://github.com/nextcloud/files_fulltextsearch>`_
 * `Full Text Search - Tesseract OCR <https://github.com/daita/files_fulltextsearch_tesseract>`_
+* `Temporary files lock <https://github.com/nextcloud/files_lock>`_ (this is necessary for "Edit files locally" functionality)
```
Comment (marcelklehr) [inline admin_manual/installation/apps_supported.rst:38]: I couldn't find anywhere where this small hint would fit better

Comment (AndyScherzinger) [reply admin_manual/installation/apps_supported.rst:38]: Seems to me to be the best place 👍

**Pull request merged**